packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.js

```diff
@@ -0,0 +1,63 @@
+import { LightningElement, api } from 'lwc';
+import { registerComponentForInit, initializeWithHeadless } from 'c/quanticHeadlessLoader';
+
+
+export default class QuanticTab extends LightningElement {
+  /** @type {import("coveo").Tab} */
+  tab;
+  /** @type {()=> void} */
+  unsubscribe;
+  /** @type {boolean} */
+  isActive = false;
+  /** @type {string} */
+  @api label;
+  /** @type {string} */
+  @api expression = "";
```

Comment:
I would leave the default as undefined instead of `""` could perhaps get us in a situation where we are trying to work with an empty string instead of realizing the expression is undefined?
```suggestion
  @api expression;
```